weaknesses/MASVS-CRYPTO/MASWE-0023.md

```diff
@@ -7,20 +7,37 @@ profiles: [L1, L2]
 mappings:
   masvs-v1: [MSTG-CRYPTO-4]
   masvs-v2: [MASVS-CRYPTO-1]
+  mastg-v1: [MASTG-TEST-0014]
 
 refs:
 - https://developer.android.com/privacy-and-security/cryptography#deprecated-functionality
 - https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-131Ar2.pdf
 - https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-38a.pdf
 - https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/TechGuidelines/TG02102/BSI-TR-02102-1.pdf?__blob=publicationFile
-draft:
-  description: The use of weak padding such as NoPadding, ZeroPadding, etc. in a security
-    sensitive context should be avoided to ensure the integrity and authenticity of
-    the data.
-  topics:
-  - NoPadding
-  - PKCS1-v1_5
-status: draft
-
 ---
 
+## Overview
+
+Outdated or weak padding schemes, such as PKCS1v1.5 or other padding schemes that fail to comply with secure standards, such as NIST SP 800-56B are not recommended for use. These padding schemes include vulnerabilities that may allow attackers to undermine security mechanisms, such as padding oracle attacks.
```

Comment:
is [NIST SP 800-56B](https://csrc.nist.gov/pubs/sp/800/56/b/r2/final) the right one? Please double check and add an inline link.

> ## Recommendation for Pair-Wise Key-Establishment Using Integer Factorization Cryptography
> 
> This Recommendation specifies key-establishment schemes using integer factorization cryptography (in particular, RSA). Both key-agreement and key transport schemes are specified for pairs of entities, and methods for key confirmation are included to provide assurance that both parties share the same keying material. In addition, the security properties associated with each scheme are provided.
>

Reply:
Also please add an inline link for "padding oracle attacks"